perfact/zodbsync/commands/upload.py

```diff
@@ -92,32 +119,41 @@ def run(self):
             for filename in files:
                 file_ending = filename.split('.')[-1]
 
-                # only support css and js files ... for now
-                if file_ending not in ['css', 'js']:
+                # bail out if not a valid extension
+                if valid_extensions and file_ending not in valid_extensions:
```

Comment:
Minor nitpick: If the argument is `--valid-extensions ""`, this is treated as if the argument were omitted completely - although I would expect that this would ignore all files instead of allowing all.
So I would like this to treat `None` differently than `[]`, just as a matter of principle.